src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -516,8 +516,22 @@ function co2_compensation(
 end
 
 """
-    rubisco_assimilation(::C3,
-                         Vcmax::FT,
+    rubisco_assimilation(is_c3::AbstractFloat, args...)
+
+Calls the correct rubisco assimilation function based on the `is_c3`.
+
+A `is_c3` value of 1.0 corresponds to C3 photosynthesis and calls
+`c3_rubisco_assimilation`, while 0.0 corresponds to C4 photsynthesis and calls
+`c4_rubisco_assimilation`. If a number between 0 and 1 is passed, if it is greater
+than 0.5, c3 will be assumed (essentially this takes the dominant mechanism)
```

Comment:
I see. I think this is fine, but Im wondering if we should think of is_c3 as represent a bool and not a number between 0 and 1 (as above).